payment_payzen/models/inherited_payment_transaction.py

```diff
@@ -98,21 +105,56 @@ def _payzen_form_get_tx_from_data(self, data):
             error_msg = "Payzen: received bad data for reference {}".format(reference)
 
             if not transactions:
-                error_msg += "; no order found"
+                error_msg += "; no payment transaction found"
             else:
-                error_msg += "; multiple order found"
+                error_msg += "; multiple payment transactions found"
 
             _logger.info(error_msg)
             raise ValidationError(error_msg)
 
-        signature = data.get('signature')
-        if signature != transactions.acquirer_id.payzen_generate_digital_sign(data):
-            error_msg = _("Payzen: signatures mismatch")
-            _logger.info(error_msg)
-            raise ValidationError(error_msg)
+        local_call = data.get('local_call', False)
+        if not local_call:
+            transactions._payzen_check_signature(data)
 
         return transactions
 
+    @api.model
+    def payzen_cron_check_draft_payment_transactions(self):
+        f"""Cron task for check all transaction in draft state and recheck is status with Payzen.
+
+        This method will get al draft payment transaction an try to get this status again (like ipn return) we can
+        pass an argument max retry number for each transaction.
```

Comment:
```suggestion
        pass an argument `max_retry_number` for each transaction.
```
Il me semble que l'on n'utilisait pas cet argument

Reply:
Effectivement cadavre du début de la création ^^.